src/intl/messages.js

```diff
@@ -669,14 +662,16 @@ export const messages: { [messageId: string]: MessageType } = {
   vmType_highPerformance: 'High Performance',
   vmType_server: 'Server',
   vncConsole: 'VNC Console',
-  vncConsoleOpen: 'Open VNC Console',
   vncConsoleBrowser: 'VNC Console (Browser)',
-  vncConsoleBrowserOpen: 'Open VNC Console (Browser)',
   vnicProfile: 'VNIC Profile',
   vnicProfileEmpty: '<Empty>',
   yearsShort: 'y',
   yes: 'Yes',
   youHaveNoAllowedVnicProfiles: 'You cannot create or edit NICs because you do not have permission to use any vNIC Profiles in the VM\'s Data Center.',
+  sortAToZ: 'Sort from A to Z',
```

Comment:
Let's keep the keys in this object sorted A-Z properly please.

Reply:
Can you you explain what you meant here?
do you mean to change the message to: "Sorted A-Z"?

Reply:
No.  The keys in the messages object are sorted alpha (i.e. A-Z) in the file.  `sortAToZ` should be in the right place, not below the "y" keys.

Reply:
Oh I've never noticed that, Done.